my.css

```diff
@@ -207,6 +212,10 @@ body {
     background: lightgray;
   }
 
+  .header__promo {
+  width: 34%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)